app/popup.js

```diff
@@ -477,6 +484,23 @@ class OptionsTab extends React.Component {
           </tbody>
         </table>
 
+        <h4 className="page-header">Import / Export</h4>
+        <div className="row">
+          <div className="col-xs-8">
+            <Button label='Export' clickHandler={_exportData} className='glyphicon-export'/>
+            <Button label='Import' clickHandler={() => {this.fileselector.click()}} className='glyphicon-import'/>
```

Comment:
This project uses [Flow](https://flow.org/), although I have not yet included it in a Gulp task. You need to declare `this.fileselector` on the `OptionsTab`:

```
fileselector: ?HTMLInputElement;
```